internal/formats/common/cyclonedxhelpers/format.go

```diff
@@ -0,0 +1,90 @@
+package cyclonedxhelpers
+
+import (
+	"time"
+
+	"github.com/CycloneDX/cyclonedx-go"
+	"github.com/anchore/syft/internal"
+	"github.com/anchore/syft/internal/version"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/sbom"
+	"github.com/anchore/syft/syft/source"
+	"github.com/google/uuid"
+)
+
+func ToFormatModel(s sbom.SBOM) *cyclonedx.BOM {
+	cdxBOM := cyclonedx.NewBOM()
+	versionInfo := version.FromBuild()
+
+	cdxBOM.SerialNumber = uuid.New().URN()
```

Comment:
I think a small comment about the specification requiring the URN format might be nice to show this was not an arbitrary selection for the SerialNumber